### ISSUE TITLE
Problem with pypi packages on remote server.

### ISSUE THREAD

**Issue by almanzor6** [tzulberti/Flask-PyPi-Proxy#10]
version of flask_pypi_proxy: 0.4.1

What happens when I run without using proxy:

```
$ pip install python-graph-core==1.8.2
Downloading/unpacking python-graph-core==1.8.2
  Downloading python-graph-core-1.8.2.tar.gz
  Running setup.py egg_info for package python-graph-core

Installing collected packages: python-graph-core
  Running setup.py install for python-graph-core
    Skipping installation of /home/w/.virtualenvs/wh-monit/lib/python2.7/site-packages/pygraph/__init__.py (namespace package)

    Installing /home/w/.virtualenvs/wh-monit/lib/python2.7/site-packages/python_graph_core-1.8.2-py2.7-nspkg.pth
Successfully installed python-graph-core
```

When I install other package using proxy:

```
$ pip install -i http://127.0.0.1:8000/simple/ furl
Downloading/unpacking furl
  Downloading furl-0.3.4.tar.gz
  Running setup.py egg_info for package furl

Downloading/unpacking orderedmultidict>=0.7.1 (from furl)
  Downloading orderedmultidict-0.7.1.tar.gz
  Running setup.py egg_info for package orderedmultidict

Installing collected packages: furl, orderedmultidict
  Running setup.py install for furl

  Found existing installation: orderedmultidict 0.7
    Uninstalling orderedmultidict:
      Successfully uninstalled orderedmultidict
  Running setup.py install for orderedmultidict

Successfully installed furl orderedmultidict
Cleaning up...
```

When I run command as below, I've got an error:

```
$pip install -i http://127.0.0.1:8000/simple/ python-graph-core==1.8.2
Downloading/unpacking python-graph-core==1.8.2
  HTTP error 404 while getting http://127.0.0.1:8000/packages/source/p/python-graph-core/python-graph-core-1.8.2.tar.gz?remote=http%3A%2F%2Fcode.google.com%2Ffiles%2Fpython-graph-core-1.8.2.tar.gz (from http://127.0.0.1:8000/simple/python-graph-core/)
  Could not install requirement python-graph-core==1.8.2 because of error HTTP Error 404: NOT FOUND
Could not install requirement python-graph-core==1.8.2 because of HTTP error HTTP Error 404: NOT FOUND for URL http://127.0.0.1:8000/packages/source/p/python-graph-core/python-graph-core-1.8.2.tar.gz?remote=http%3A%2F%2Fcode.google.com%2Ffiles%2Fpython-graph-core-1.8.2.tar.gz (from http://127.0.0.1:8000/simple/python-graph-core/)
```

In log file we can find:

```
2013-06-29 11:42:23,500 [DEBUG] Requesting index for: python-graph-core
2013-06-29 11:42:23,500 [DEBUG] Didnt found package: python-graph-core in local repository. Using proxy.
2013-06-29 11:42:23,520 [INFO] Starting new HTTP connection (1): pypi.python.org
2013-06-29 11:42:28,836 [DEBUG] "GET /simple/python-graph-core HTTP/1.1" 301 0
2013-06-29 11:42:28,902 [DEBUG] "GET /simple/python-graph-core/ HTTP/1.1" 200 264
2013-06-29 11:42:28,911 [INFO] Starting new HTTP connection (1): code.google.com
2013-06-29 11:42:29,505 [DEBUG] "GET /p/python-graph/downloads/list?can=1 HTTP/1.1" 200 None
2013-06-29 11:42:29,996 [INFO] Starting new HTTP connection (1): code.google.com
2013-06-29 11:42:30,464 [DEBUG] "GET /p/python-graph/downloads/list HTTP/1.1" 200 None
2013-06-29 11:42:30,612 [DEBUG] Downloading: python-graph-core-1.8.2.tar.gz
2013-06-29 11:42:30,612 [DEBUG] Starting to download: python-graph-core-1.8.2.tar.gz using the url: http://code.google.com/files/python-graph-core-1.8.2.tar.gz
2013-06-29 11:42:30,614 [INFO] Starting new HTTP connection (1): code.google.com
2013-06-29 11:42:30,749 [DEBUG] "GET /files/python-graph-core-1.8.2.tar.gz HTTP/1.1" 404 967
2013-06-29 11:42:30,751 [DEBUG] Finished downloading package: python-graph-core-1.8.2.tar.gz
2013-06-29 11:42:30,751 [WARNING] Error respose while downloading for proxy: python-graph-core-1.8.2.tar.gzResponse details: <!DOCTYPE html>
<html lang=en>
  <meta charset=utf-8>
  <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">
  <title>Error 404 (Not Found)!!1</title>
  <style>
    *{margin:0;padding:0}html,code{font:15px/22px arial,sans-serif}html{background:#fff;color:#222;padding:15px}body{margin:7% auto 0;max-width:390px;min-height:180px;padding:30px 0 15px}* > body{background:url(//www.google.com/images/errors/robot.png) 100% 5px no-repeat;padding-right:205px}p{margin:11px 0 22px;overflow:hidden}ins{color:#777;text-decoration:none}a img{border:0}@media screen and (max-width:772px){body{background:none;margin-top:0;max-width:none;padding-right:0}}
  </style>
  <a href=//www.google.com/><img src=//www.google.com/images/errors/logo_sm.gif alt=Google></a>
  <p><b>404.</b> <ins>That’s an error.</ins>
  <p>The requested URL <code>/files/python-graph-core-1.8.2.tar.gz</code> was not found on this server.  <ins>That’s all we know.</ins>
```

Is this a bug or is there anything wrong with my configuration?

**Comment by tzulberti** [comment]
Its seems to be a bug...

**Comment by tzulberti** [comment]
There was a problem when parsing the google.com URLs. This issue is solved on new release version 0.4.2.

Thanks,
Tomas Zulberti
